recipes/mcxtrace-suite/activate.sh

```diff
@@ -0,0 +1,2 @@
+export MCXTRACE="${CONDA_PREFIX}/share/mcxtrace/resources"
```

Comment:
It is not a big deal but, if you want to cover cases where a user may legitimately have that env var defined before using this package, you should:

1. check if it exists
2. store its value
3. define the new one
4. then reverse this in the deactivation

See https://github.com/conda-forge/gdal-feedstock/blob/main/recipe/scripts/activate.sh for an example.

Again, not mandatory to merge. This is mostly how far do you want to go to ensure a smooth experience to user in possible corner cases.